web/spec/supabase.yml

```diff
@@ -565,6 +571,8 @@ pages:
   auth.user():
     title: 'user()'
     $ref: '@supabase/gotrue-js.GoTrueClient.user'
+    notes: |
+      This method gets the user object stored in local storage.
```

Comment:
@kangmingtay - this part is technically not true, as it gets it from memory (eg, `this.user`). LocalStorage is actually optional when initiating the client (or a dev can provider their own storage)

Reply:
you're right! (https://github.com/supabase/gotrue-js/blob/master/src/GoTrueClient.ts#L296-L298) haha will make the update